src/matrix_cookbook/for_mathlib/linear_algebra/matrix/adjugate.lean

```diff
@@ -17,28 +17,28 @@ import matrix_cookbook.for_mathlib.data.matrix
 open matrix
 open_locale matrix big_operators
 
-variables {R: Type*}[comm_ring R]
+variables {R : Type*} [comm_ring R]
 
-def cofactor {n : ℕ} (A : matrix (fin n.succ) (fin n.succ) R) (i j: fin n.succ)  :=
-  (-1)^(i + j : ℕ) * det (A.submatrix i.succ_above j.succ_above)
-
-/- Lemma should state
-lemma eq_146 (n : ℕ) (A : matrix (fin n.succ) (fin n.succ) ℂ) (i j) :
-  adjugate A j i = cofactor A i j := 
--/
-lemma adjugate_eq_cofactor_transpose {n : ℕ} (A : matrix (fin n.succ) (fin n.succ) R) 
-  (i j) : adjugate A j i = cofactor A i j := 
+lemma adjugate_eq_det_submatrix {n : ℕ} (A : matrix (fin n.succ) (fin n.succ) R) (i j) :
+  adjugate A i j = (-1)^(j + i : ℕ) * det (A.submatrix j.succ_above i.succ_above) := 
 begin
-  rw adjugate, dsimp,
-  rw cramer_transpose_apply,
-  rw det_succ_row _ i,
-  conv_lhs { apply_congr, skip, rw update_row_apply, },
-  simp only [eq_self_iff_true, if_true],
-  conv_lhs {apply_congr, skip, rw pi.single_apply j (1:R) x, 
-  rw mul_ite, rw ite_mul, rw mul_zero, rw zero_mul, },
-  simp only [mul_one, finset.sum_ite_eq', finset.mem_univ, if_true, 
-    neg_one_pow_mul_eq_zero_iff],
-  rw cofactor,
-  rw submatrix_update_row_succ_above,
+  simp_rw [adjugate_apply, det_succ_row _ j, update_row_self, submatrix_update_row_succ_above],
+  rw [fintype.sum_eq_single i (λ h hjk, _), pi.single_eq_same, mul_one],
+  rw [pi.single_eq_of_ne hjk, mul_zero, zero_mul],
```

Comment:
The main trick you missed here is that `simp_rw` saves you the need to do `conv` manually.

The other important trick is to use `fintype.sum_eq_single` which deals with all but one of the terms vanishing for you.